### ISSUE TITLE
Use of string.match with is NYI in LuaJIT

### ISSUE THREAD

**Issue by edo888** [openresty/lua-resty-memcached#28]
Hi,

I have noticed that this library uses [string.match](https://github.com/openresty/lua-resty-memcached/blob/d4b0186d5e2f65e38c3c63408d462d98c8dc5977/lib/resty/memcached.lua#L6) function which is not supported by LuaJIT: http://wiki.luajit.org/NYI#libraries_string-library

Will this library play nice with LuaJIT or there are other obstacles which I'm not aware of?

Thanks!

**Comment by agentzh** [comment]
@edo888 Yeah, we should use `ngx.re.find` or `ngx.re.match` for it. Pull requests welcome!
